tests/auto/integration/tst_integration.cpp

```
@@ -124,6 +134,15 @@ class tst_integration : public QObject {
     pass.updateEnv();
   }
 
+  static QByteArray gpgInsertErrorMsg(const QSignalSpy &errorSpy) {
```

Clang (14.0.6)

[error] use a trailing return type for this function

(modernize-use-trailing-return-type,-warnings-as-errors)